bindings/py/cpp_src/bindings/algorithms/py_SpatialPooler.cpp

```diff
@@ -186,87 +185,87 @@ using namespace sdr;
 	    ); 
 
         // setBoostFactors
-        py_SpatialPooler.def("setBoostFactors", [](SpatialPooler& self, py::array_t<Real>& x)
+        py_SpatialPooler.def("setBoostFactors", [](SpatialPooler& self, py::array& x)
```

Comment:
all py bindings now use py::array for the reasons @fcr found in #484

Reply:
I thought that they should have been changed as you did but I didn't want to make too many waves. :-)